### PR TITLE
Improve docs on DESCRIBE and EXPLAIN

### PR DESCRIPTION
**Pull request by miguno**

### CODE REVIEW

File: docs/syntax-reference.md [hunk 1]
```diff
@@ -337,6 +226,121 @@ The WITH clause supports the following properties:
 | TIMESTAMP               | Associates the message timestamp in the Kafka topic with a column in the KSQL table. Time-based operations such as windowing will process a record according to this timestamp. |
 
 
+### DESCRIBE
```
Comment (miguno) [inline docs/syntax-reference.md]: Moved _after_ the CREATE statements as discussed offline.

File: docs/syntax-reference.md [hunk 2]
```diff
@@ -337,6 +226,121 @@ The WITH clause supports the following properties:
...
+
+**Synopsis**
+
+```sql
+DESCRIBE [EXTENDED] (stream_name|table_name);
```
Comment (miguno) [inline docs/syntax-reference.md]: I collapsed DESCRIBE and DESCRIBE EXTENDED to make it easier to see it's "one" statement.

File: docs/syntax-reference.md [hunk 3]
```diff
@@ -337,6 +226,121 @@ The WITH clause supports the following properties:
...
+```
+
+
+**Description**
+
+* DESCRIBE: List the columns in a stream or table along with their data type and other attributes.
+* DESCRIBE EXTENDED: Display DESCRIBE information with additional runtime statistics, Kafka topic details, and the
```
Comment (miguno) [inline docs/syntax-reference.md]: An intentional side effect of this bullet list is that "DESCRIBE EXTENDED" (unlike "DESCRIBE [EXTENDED]" in the synopsis) can be found via `Ctrl-F`-ing the document. ;-)

**Comment by joel-hamill** [comment]
@miguno please make this same change in the KSQL CP branch too https://github.com/confluentinc/ksql/tree/joel-hamill/merge-ksql-2-cp

**Comment by miguno** [comment]
@joel-hamill : See https://github.com/confluentinc/ksql/pull/547 for the CP docs.

**Pull request merged**